public/index.html

```diff
@@ -38,5 +38,6 @@
       To begin the development, run `npm start`.
       To create a production bundle, use `npm run build`.
     -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>  
```

Comment:
Is this required ?

Reply:
@akshatnitd , I forgot to remove this line when I removed bootstrap , I've removed it now,have a look